configure.js

```diff
@@ -1,10 +1,17 @@
-var ipc = require('ipc')
 var Ractive = require('ractive')
 var page = require('page')
 var fs = require('fs')
+var Client = require('electron-rpc/client')
+var client = new Client()
```

Comment:
If we want to use more features, we can also start a whole express server on a random port. But currently there's no need for it.